aws-lc-fips-sys/CMakeLists.txt

```diff
@@ -60,4 +60,8 @@ if (BORINGSSL_PREFIX)
         set(TARGET_PREFIX "lib${BORINGSSL_PREFIX}")
     endif()
     set_my_target_properties(PREFIX ${TARGET_PREFIX})
+
+    # This BORINGSSL_PREFIX has an "_" appended, so we must remove it
+    string(REGEX REPLACE "_$" "" BORINGSSL_PREFIX_MACRO ${BORINGSSL_PREFIX})
+    add_definitions(-DBORINGSSL_PREFIX=${BORINGSSL_PREFIX_MACRO})
```

Comment:
`set(BORINGSSL_PREFIX_MACRO "${BORINGSSL_PREFIX}")` and then using `BORINGSSL_PREFIX_MACRO` for the in/out to the regex replace might be clearer.